test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -599,20 +600,21 @@ define([
                 await this.jobStatusTableInstance.start({
                     node: container,
                 });
-                // this table already has the info populated, so the 'jobStatusTable-info'
-                // handler is not required
                 handlers.forEach((handler) => {
                     const { event, name } = handler;
-                    if (event === 'job-info') {
-                        expect(this.jobManager.handlers[event][name]).not.toBeDefined();
-                    } else {
-                        expect(this.jobManager.handlers[event][name]).toBeDefined();
-                    }
```

Comment:
Job info handler always gets added, so alter these tests slightly; check that appropriate listeners have been added instead.